shared/pinentry/remote-proxy.desktop.tsx

```diff
@@ -1,86 +1,44 @@
 // A mirror of the remote pinentry windows.
 // RemotePinentrys renders all of them (usually only one)
 // RemotePinentry is a single remote window
-import * as I from 'immutable'
+import * as RPCTypes from '../constants/types/rpc-gen'
 import * as React from 'react'
 import * as Styles from '../styles'
-import * as Types from '../constants/types/pinentry'
 import SyncProps from '../desktop/remote/sync-props.desktop'
 import SyncBrowserWindow from '../desktop/remote/sync-browser-window.desktop'
-import {NullComponent, connect, mapProps, compose} from '../util/container'
+import * as Container from '../util/container'
+import * as Types from '../constants/types/pinentry'
 import {serialize} from './remote-serializer.desktop'
 
-const dataToProps = mapProps(
-  ({
-    data,
-    remoteWindowNeedsProps,
-    darkMode,
-  }: {
-    darkMode: boolean
-    data: Types.PinentryState
-    remoteWindowNeedsProps: number
-  }) => ({
-    cancelLabel: data.cancelLabel,
-    darkMode,
-    prompt: data.prompt,
-    remoteWindowNeedsProps,
-    retryLabel: data.retryLabel,
-    sessionID: data.sessionID,
-    showTyping: data.showTyping,
-    submitLabel: data.submitLabel,
-    submitted: data.submitted,
-    type: data.type,
-    windowComponent: 'pinentry',
-    windowOpts: {height: 210, width: 440},
-    windowParam: String(data.sessionID),
-    windowPositionBottomRight: false,
-    windowTitle: 'Pinentry',
-  })
-)
+export type WireProps = {
```

Comment:
the props we send across the wire. before it was mostly any all over and now we can use this to better tie things together